org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/completion/LSContentAssistProcessor.java

```diff
@@ -289,17 +309,17 @@ public IContextInformation[] computeContextInformation(ITextViewer viewer, int o
 		List<IContextInformation> contextInformations = Collections.synchronizedList(new ArrayList<>());
 		try {
 			this.contextInformationLanguageServersFuture = LanguageServers.forDocument(document)
-					.withFilter(capabilities -> capabilities.getSignatureHelpProvider() != null).collectAll(
-							ls -> ls.getTextDocumentService().signatureHelp(param).thenAccept(signatureHelp -> {
-								if (signatureHelp != null) {
-									signatureHelp.getSignatures().stream()
-											.map(LSContentAssistProcessor::toContextInformation)
-											.forEach(contextInformations::add);
-								}
-							}));
+					.withFilter(capabilities -> capabilities.getSignatureHelpProvider() != null)
+					.collectAll(ls -> ls.getTextDocumentService().signatureHelp(param).thenAccept(signatureHelp -> {
+						if (signatureHelp != null) {
+							signatureHelp.getSignatures().stream().map(LSContentAssistProcessor::toContextInformation)
+									.forEach(contextInformations::add);
+						}
+					}));
 			this.contextInformationLanguageServersFuture.get(CONTEXT_INFORMATION_TIMEOUT, TimeUnit.MILLISECONDS);
 		} catch (ResponseErrorException | ExecutionException e) {
-			if (!CancellationUtil.isRequestCancelledException(e)) { // do not report error if the server has cancelled the request
+			if (!CancellationUtil.isRequestCancelledException(e)) { // do not report error if the server has cancelled
```

Comment:
the indentation of this comment is a bit weird now (at least on the GitHub review. Could we revert the change, or make it shorter so that the auto-format leaves it all in one line?

Reply:
I could, but this is done by the current Eclipse formatter settings and I prefer to use the formatter settings because otherwise we have to manually update readjust the code everytime when we do code changes and use the formatter.

Reply:
I would actually really appreciate if we could harmonize the formatter settings of all LSP4E modules (they currently differ) and in a single commit reformat the whole project with these settings.